pyworklistserver/worklist.py

```diff
@@ -95,7 +94,7 @@ def get_random_worklist(self):
         worklist_item = Dataset()
         worklist_item.StudyInstanceUID = generate_uid(prefix='1.2.840.113619.2.391.6789.', entropy_srcs=[_random_unicode_string(10, _get_random_language_string()), _random_unicode_string(10, _get_random_language_string())])
         worklist_item.Modality = 'US'
-        worklist_item.SpecificCharacterSet = self._specific_charset
+        worklist_item.SpecificCharacterSet = ['ISO_IR 100', 'ISO_IR 144', 'UTF8', 'ISO_IR 126', 'ISO_IR 13', 'ISO 2022 IR 149']
```

Comment:
Please explain what it means to provide many character sets. I thought a worklist could only contain a single character set.

Reply:
Thats what we thought to. Until we read in the documentation that the specificcharacterset could have a multiplisity 1-n.